report/thermodesorption-spectra/ogorodnikova-tds.ipynb

```diff
@@ -0,0 +1,299 @@
+      "  warnings.warn(\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "100.0 %        5.0e+02 s    Ellapsed time so far: 10.3 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "import festim as F\n",
+    "import numpy as np\n",
+    "import sympy as sp\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "flux_model = F.Simulation()\n",
+    "\n",
+    "vertices = np.concatenate([\n",
+    "    np.linspace(0, 30e-9, num=200),\n",
+    "    np.linspace(30e-9, 3e-6, num=300),\n",
+    "    np.linspace(3e-6, 20e-6, num=200),\n",
+    "])\n",
+    "\n",
+    "flux_model.mesh = F.MeshFromVertices(vertices)\n",
+    "# Material Setup, only W\n",
+    "#　Data from Table 1\n",
+    "tungsten = F.Material(\n",
+    "    id=1,\n",
+    "    D_0=4.1e-07,  # m2/s\n",
+    "    E_D=0.39,  # eV\n",
+    ")\n",
+    "\n",
+    "flux_model.materials = tungsten\n",
+    "import sympy as sp\n",
+    "\n",
+    "imp_fluence = 1e22\n",
+    "flux_strength = 2.5e19 # beam strength from paper\n",
+    "\n",
+    "imp_time = imp_fluence/flux_strength # s\n",
+    "\n",
+    "ion_flux = sp.Piecewise((flux_strength, F.t <= imp_time), (0, True))\n",
+    "\n",
+    "source_term = F.ImplantationFlux(\n",
+    "    flux=ion_flux,  # H/m2/s\n",
+    "    imp_depth=4.5e-9,  # m\n",
+    "    width=2.5e-9,  # m\n",
+    "    volume=1\n",
+    ")\n",
+    "\n",
+    "flux_model.sources = [source_term]\n",
+    "#trap settings\n",
+    "w_atom_density = 6.3e28  # atom/m3\n",
+    "\n",
+    "trap_1 = F.Trap(\n",
+    "        k_0=4.1e-7/(1.1e-10**2*6*w_atom_density),\n",
+    "        E_k=0.39,\n",
+    "        p_0=1e13,\n",
+    "        E_p=0.87,\n",
+    "        density=1.3e-3*w_atom_density,\n",
+    "        materials=tungsten\n",
+    "    )\n",
+    "\n",
+    "trap_2 = F.Trap(\n",
+    "        k_0=4.1e-7/(1.1e-10**2*6*w_atom_density),\n",
+    "        E_k=0.39,\n",
+    "        p_0=1e13,\n",
+    "        E_p=1.0,\n",
+    "        density=4e-4*w_atom_density,\n",
+    "        materials=tungsten\n",
+    "    )\n",
+    "\n",
+    "center = 4.5e-9\n",
+    "width = 2.5e-9\n",
+    "distribution = 1/(width*(2*sp.pi)**0.5) * sp.exp(-0.5*((F.x-center)/width)**2)\n",
+    "\n",
+    "trap_3 = F.ExtrinsicTrap(\n",
+    "        k_0=4.1e-7/(1.1e-10**2*6*w_atom_density),\n",
+    "        E_k=0.39,\n",
+    "        p_0=1e13,\n",
+    "        E_p=1.5,\n",
+    "        phi_0=ion_flux,\n",
+    "        n_amax=1e-01*w_atom_density, f_a=distribution, eta_a=6e-4,\n",
+    "        n_bmax=1e-02*w_atom_density, f_b=sp.Piecewise((1e6, F.x < 1e-6), (0, True)), eta_b=2e-4,\n",
+    "        materials=tungsten\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "center = 4.5e-9\n",
+    "width = 2.5e-9\n",
+    "distribution = 1/(width*(2*sp.pi)**0.5) * sp.exp(-0.5*((F.x-center)/width)**2)\n",
```

Comment:
Looks like these parameters are declared twice.

Reply:
Yes, `center`, `width`, and `distribution` are declared twice